File: kotlin-spring/src/main/kotlin/traveltimeplatform/models/ResponseTimeFilterPostcodeDistrictProperties.kt
package traveltimeplatform.models

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import traveltimeplatform.models.ResponseTravelTimeStatistics
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param travelTimeReachable 
 * @param travelTimeAll 
 * @param coverage 
 */
data class ResponseTimeFilterPostcodeDistrictProperties (

        @JsonProperty("travel_time_reachable") val travelTimeReachable: ResponseTravelTimeStatistics? = null,

        @JsonProperty("travel_time_all") val travelTimeAll: ResponseTravelTimeStatistics? = null,

        @JsonProperty("coverage") val coverage: Double? = null
) {

}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseTimeFilterPostcodeDistrictProperties.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.ResponseTravelTimeStatistics

module ResponseTimeFilterPostcodeDistrictProperties = 

  //#region ResponseTimeFilterPostcodeDistrictProperties


  type ResponseTimeFilterPostcodeDistrictProperties = {
    TravelTimeReachable : ResponseTravelTimeStatistics;
    TravelTimeAll : ResponseTravelTimeStatistics;
    Coverage : double;
  }
  //#endregion
  